crates/swc/tests/tsc-references/stringNamedPropertyAccess.2.minified.js

```diff
@@ -1,7 +1,4 @@
 //// [stringNamedPropertyAccess.ts]
+var i, a;
 import _class_call_check from "@swc/helpers/src/_class_call_check.mjs";
-var i, a, C = function C() {
```

Comment:
I'm not sure if this is correct.
This file calls `_class_call_check` but the new version does not call it

Reply:
Oh is there `PURE`?

Reply:
No, it got inlined into
```js
(function C() {
    "use strict";
    _class_call_check(this, C);
})["c d"]
```
as access to function property is pure, this got elimintated.

Reply:
I see, thank you!